lib/dao.ts

```diff
@@ -48,9 +48,9 @@ export class DAO {
     return new Promise<string>(
       async (resolve: (address: Address) => void, reject: (ex: any) => void): Promise<void> => {
         try {
-          const contracts = await Contracts.getDeployedContracts();
-          const daoCreator = await DaoCreator.at(
-            daoCreatorAddress ? daoCreatorAddress : contracts.allContracts.DaoCreator.address);
+          const daoCreator =
+            daoCreatorAddress ?
+              await WrapperService.factories.DaoCreator.at(daoCreatorAddress) : WrapperService.wrappers.DaoCreator;
```

Comment:
this still feels pretty confusing and non intuitive to me. hmm. im fine with this refactor but i think there is a next level of clarity we can and should get to

Reply:
Something like this might be good:

```
const daoCreator = daoCreatorAddress ? 
   await ContractWrappers.DaoCreator.factory.at(daoCreatorAddress) : 
   ContractWrappers.DaoCreator; 
```